scripts/Util.py

```diff
@@ -748,11 +752,16 @@ def getProps(self, process, current):
                     props["Ice.Override.Compress"] = "1"
                 if self.serialize:
                     props["Ice.ThreadPool.Server.Serialize"] = "1"
-                props["Ice.IPv6"] = self.ipv6
+                # TODO: Remove once this property is removed in all mappings
+                if not isinstance(process.getMapping(current), CSharpMapping):
+                    props["Ice.IPv6"] = self.ipv6
                 if self.ipv6:
                     props["Ice.PreferIPv6Address"] = True
                 if self.mx:
-                    props["Ice.Admin.Endpoints"] = "tcp -h \"::1\"" if self.ipv6 else "tcp -h 127.0.0.1"
+                    # TODO: the admin endpoints are also necessary for IceGrid/simple test.
+                    # Do we ever run it without mx ?
```

Comment:
I'm not sure I understand the TODO here. Yes the IceGrid/simple test doesn't necessarily run with the Ice admin enabled, only if --mx is set.

Reply:
If `mx` is not enabled then IceGrid testing won't work